karaf/features/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features xmlns="http://karaf.apache.org/xmlns/features/v1.4.0" name="${project.groupId}-${project.version}">
+
+    <feature name="brooklyn-tosca" version="${project.version}">
+        <bundle>mvn:io.cloudsoft.brooklyn.tosca/brooklyn-tosca-karaf-init/${project.version}</bundle>
+        <bundle dependency="true">mvn:org.apache.brooklyn/brooklyn-launcher-common/${brooklyn.version}</bundle>
```

Comment:
It _is_ there to make Karaf happy but can't see how to put the dependency elsewhere. Even if we could remove it I think it would be worth leaving the feature in place for consistency's sake (and if nothing else it's easier to type the feature-install command than the bundle-install one!)